{
    "output": {
        "SOltkC3J2I2025-05-2818:47:33.139942": {
            "Lz4ZaPp": "\nUser accounts for \\\\OMER\n\n-------------------------------------------------------------------------------\nAdministrator            DefaultAccount           Guest                    \nhacker                   WDAGUtilityAccount       \nThe command completed successfully.\n\n",
            "PVlKUSA": " Volume in drive C is TI31219900B\n Volume Serial Number is FEF1-A394\n\n Directory of C:\\Users\\hacker\\PycharmProjects\\backdoor\\GhostTrigger\n\n05/28/2025  06:47 PM    <DIR>          .\n05/28/2025  06:47 PM    <DIR>          ..\n05/26/2025  09:41 PM            17,895 GhostTrigger.py\n05/28/2025  06:47 PM            12,288 info.db\n05/29/2025  03:01 PM            16,285 PhantomGate.py\n05/21/2025  10:00 PM             1,932 test.py\n               4 File(s)         48,400 bytes\n               2 Dir(s)  48,735,858,688 bytes free\n",
            "mB8FFwf": "\f",
            "GKM7Cbf": "'ls' is not recognized as an internal or external command,\noperable program or batch file.\n",
            "AWkGBja": "'pwd' is not recognized as an internal or external command,\noperable program or batch file.\n",
            "ceQx8TV": "'lslqdk' is not recognized as an internal or external command,\noperable program or batch file.\n",
            "fkivv7L": " Volume in drive C is TI31219900B\n Volume Serial Number is FEF1-A394\n\n Directory of C:\\Users\\hacker\\PycharmProjects\\backdoor\\GhostTrigger\n\n05/28/2025  06:47 PM    <DIR>          .\n05/28/2025  06:47 PM    <DIR>          ..\n05/26/2025  09:41 PM            17,895 GhostTrigger.py\n05/28/2025  06:47 PM            12,288 info.db\n05/29/2025  03:39 PM            16,165 PhantomGate.py\n05/21/2025  10:00 PM             1,932 test.py\n               4 File(s)         48,280 bytes\n               2 Dir(s)  48,731,967,488 bytes free\n",
            "FjS8Icq": "'ps' is not recognized as an internal or external command,\noperable program or batch file.\n",
            "xEJVnNs": "'ls' is not recognized as an internal or external command,\noperable program or batch file.\n",
            "mcloqoU": " Volume in drive C is TI31219900B\n Volume Serial Number is FEF1-A394\n\n Directory of C:\\Users\\hacker\\PycharmProjects\\backdoor\\GhostTrigger\n\n05/28/2025  06:47 PM    <DIR>          .\n05/28/2025  06:47 PM    <DIR>          ..\n05/26/2025  09:41 PM            17,895 GhostTrigger.py\n05/28/2025  06:47 PM            12,288 info.db\n05/29/2025  03:39 PM            16,165 PhantomGate.py\n05/21/2025  10:00 PM             1,932 test.py\n               4 File(s)         48,280 bytes\n               2 Dir(s)  48,967,880,704 bytes free\n",
            "ZkClrP1": " Volume in drive C is TI31219900B\n Volume Serial Number is FEF1-A394\n\n Directory of C:\\Users\\hacker\\PycharmProjects\\backdoor\n\n06/11/2025  03:31 PM    <DIR>          .\n06/11/2025  03:31 PM    <DIR>          ..\n06/08/2025  06:14 PM    <DIR>          .env\n04/18/2025  09:48 PM                66 .gitattributes\n04/18/2025  09:48 PM             3,238 .gitignore\n06/11/2025  03:31 PM    <DIR>          GhostTrigger\n03/10/2025  02:18 AM            77,474 image0_0.jpg\n03/10/2025  02:08 AM           101,564 image1_0.jpg\n05/28/2025  06:47 PM            12,288 info.db\n04/18/2025  09:48 PM             1,084 LICENSE\n06/10/2025  08:34 PM             4,456 main.py\n06/08/2025  07:10 PM            12,288 people.db\n06/09/2025  05:28 AM             3,363 README.md\n06/08/2025  07:24 PM                 0 setting.py\n06/08/2025  08:02 PM             3,692 terminal.py\n              11 File(s)        219,513 bytes\n               4 Dir(s)  13,358,612,480 bytes free\n",
            "ZF7Mg85": " Volume in drive C is TI31219900B\n Volume Serial Number is FEF1-A394\n\n Directory of C:\\Users\\hacker\\PycharmProjects\\backdoor\n\n06/11/2025  03:31 PM    <DIR>          .\n06/11/2025  03:31 PM    <DIR>          ..\n06/08/2025  06:14 PM    <DIR>          .env\n04/18/2025  09:48 PM                66 .gitattributes\n04/18/2025  09:48 PM             3,238 .gitignore\n06/11/2025  03:31 PM    <DIR>          GhostTrigger\n03/10/2025  02:18 AM            77,474 image0_0.jpg\n03/10/2025  02:08 AM           101,564 image1_0.jpg\n05/28/2025  06:47 PM            12,288 info.db\n04/18/2025  09:48 PM             1,084 LICENSE\n06/10/2025  08:34 PM             4,456 main.py\n06/08/2025  07:10 PM            12,288 people.db\n06/09/2025  05:28 AM             3,363 README.md\n06/08/2025  07:24 PM                 0 setting.py\n06/08/2025  08:02 PM             3,692 terminal.py\n              11 File(s)        219,513 bytes\n               4 Dir(s)  13,358,612,480 bytes free\n",
            "pijWC70": " Volume in drive C is TI31219900B\n Volume Serial Number is FEF1-A394\n\n Directory of C:\\Users\\hacker\\PycharmProjects\\backdoor\n\n06/11/2025  03:31 PM    <DIR>          .\n06/11/2025  03:31 PM    <DIR>          ..\n06/08/2025  06:14 PM    <DIR>          .env\n04/18/2025  09:48 PM                66 .gitattributes\n04/18/2025  09:48 PM             3,238 .gitignore\n06/11/2025  03:31 PM    <DIR>          GhostTrigger\n03/10/2025  02:18 AM            77,474 image0_0.jpg\n03/10/2025  02:08 AM           101,564 image1_0.jpg\n05/28/2025  06:47 PM            12,288 info.db\n04/18/2025  09:48 PM             1,084 LICENSE\n06/10/2025  08:34 PM             4,456 main.py\n06/08/2025  07:10 PM            12,288 people.db\n06/09/2025  05:28 AM             3,363 README.md\n06/08/2025  07:24 PM                 0 setting.py\n06/08/2025  08:02 PM             3,692 terminal.py\n              11 File(s)        219,513 bytes\n               4 Dir(s)  13,358,612,480 bytes free\n",
            "BSQCxJH": "'ls' is not recognized as an internal or external command,\noperable program or batch file.\n",
            "ZIlV5X7": "'lsllll' is not recognized as an internal or external command,\noperable program or batch file.\n",
            "CC3MNLK": " Volume in drive C is TI31219900B\n Volume Serial Number is FEF1-A394\n\n Directory of C:\\Users\\hacker\\PycharmProjects\\backdoor\n\n06/11/2025  03:31 PM    <DIR>          .\n06/11/2025  03:31 PM    <DIR>          ..\n06/08/2025  06:14 PM    <DIR>          .env\n04/18/2025  09:48 PM                66 .gitattributes\n04/18/2025  09:48 PM             3,238 .gitignore\n06/11/2025  03:31 PM    <DIR>          GhostTrigger\n03/10/2025  02:18 AM            77,474 image0_0.jpg\n03/10/2025  02:08 AM           101,564 image1_0.jpg\n05/28/2025  06:47 PM            12,288 info.db\n04/18/2025  09:48 PM             1,084 LICENSE\n06/10/2025  08:34 PM             4,456 main.py\n06/08/2025  07:10 PM            12,288 people.db\n06/09/2025  05:28 AM             3,363 README.md\n06/08/2025  07:24 PM                 0 setting.py\n06/08/2025  08:02 PM             3,692 terminal.py\n              11 File(s)        219,513 bytes\n               4 Dir(s)  13,355,048,960 bytes free\n",
            "ZGqi8AF": "'ls' is not recognized as an internal or external command,\noperable program or batch file.\n"
        }
    },
    "user-info": {},
    "target-info": {
        "SOltkC3J2I2025-05-2818:47:33.139942": {
            "ip": "172.18.144.1",
            "os": "Windows"
        },
        "kmwMqamfiF2025-06-1115:27:48.411195": {
            "ip": "172.18.144.1",
            "os": "Windows"
        }
    },
    "socket-status": {}
}